frontend/src/framework/ApiService.ts

```diff
@@ -5,6 +5,7 @@ import { makeUrl, urlToString } from "./utils/url";
 class ApiService extends ApiServiceBase {
     constructor() {
         const apiConfig = {
+            BASE: "/api",
```

Comment:
You could remove this line here (BASE), since it’s already set to ‘/api’ by default.